index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+
+<!-- todo: DRY with moqo2gbfs/publish-barshare-gbfs/index.html -->
+
+<html lang="de">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>experimenteller GTFS-Flex-Feed des VBB</title>
+	<style type="text/css">
```

Comment:
Shouldn't we provide some kind of standard style for opendata.bbnavi datasets and include the css?

Reply:
Ah, I see DRY... ok then for now